include/xrpl/protocol/UintTypes.h

```diff
@@ -46,12 +46,6 @@ class NodeIDTag
     explicit NodeIDTag() = default;
 };
 
-class MPTTag
-{
-public:
-    explicit MPTTag() = default;
-};
-
```

Comment:
I'll leave a comment here, but around line 58 there's the following code:

```
 /** MPT is a 192-bit hash representing MPTID. */
 using MPT = std::pair<std::uint32_t, AccountID>;
 ```
For better type safety, consider making a class for MPT. I'm also not sure this is the best place for this declaration.

Also, around line 69 is the following declaration:
```
 MPT const&
 noMPT();
```
It would be better if we didn't need these sentinal classes. What is the purpose of this? I don't see it used.